test/test_meta.py

```diff
@@ -1325,6 +1454,53 @@ def test_embedding_bag_byte_unpack(self):
             self.assertEqual(res.dtype, torch.float32)
             self.assertEqual(res.untyped_storage().data_ptr(), 0)
 
+    def test_meta_function_with_out(self):
```

Comment:
This test has now been superseded by the one above that tests on all PyTorch, so we can just remove it.